Android/Guava/source/Guava/Guava.csproj

```diff
@@ -1,64 +1,81 @@
-    <DebugType>full</DebugType>
-    <Optimize>true</Optimize>
-    <OutputPath>bin\Release</OutputPath>
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
-    <AndroidUseSharedRuntime>false</AndroidUseSharedRuntime>
-    <ConsolePause>false</ConsolePause>
+
+  <PropertyGroup>
+    <PackageId>Xamarin.Google.Guava</PackageId>
+    <Title>Google Guava reference library for Xamarin.Android</Title>
+    <Summary>Xamarin.Android bindings for Google Guava</Summary>
+    <Description>
+    NOTE: This package is meant to be used as a reference only for other binding projects that depend on Guava.
+    This package does not expose any useful API's in the .dll itself.
+    
+    Guava is a set of core libraries that includes new collection types (such as multimap and multiset), immutable collections, a graph library, functional types, an in-memory cache, and APIs/utilities for concurrency, I/O, hashing, primitives, reflection, string processing, and much more!
+    </Description>
+    <Authors>Microsoft</Authors>
+    <Owners>Microsoft</Owners>
+    <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
+    <PackageProjectUrl>https://go.microsoft.com/fwlink/?linkid=865028</PackageProjectUrl>
+    <PackageLicenseUrl>https://go.microsoft.com/fwlink/?linkid=865030</PackageLicenseUrl>
+    <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
+    <PackageVersion>27.1</PackageVersion>
   </PropertyGroup>
+
   <ItemGroup>
-    <Reference Include="System" />
-    <Reference Include="System.Xml" />
-    <Reference Include="System.Core" />
-    <Reference Include="Mono.Android" />
+    <TransformFile Include="Transforms\Metadata.xml" />
   </ItemGroup>
+
   <ItemGroup>
-    <Compile Include="Properties\AssemblyInfo.cs" />
+    <None Include="..\..\externals\guava.jar">
+      <Link>guava.jar</Link>
+    </None>
+    <None Include="..\..\externals\animal-sniffer-annotations.jar">
+      <Link>animal-sniffer-annotations.jar</Link>
+    </None>
+    <None Include="..\..\externals\checker-compat-qual-annotations.jar">
+      <Link>checker-compat-qual-annotations.jar</Link>
+    </None>
+    <None Include="..\..\externals\error-prone-annotations.jar">
+      <Link>error-prone-annotations.jar</Link>
+    </None>
+    <None Include="..\..\externals\j2objc-annotations.jar">
+      <Link>j2objc-annotations.jar</Link>
+    </None>
+    <None Include="..\..\externals\jsr305-annotations.jar">
+      <Link>jsr305-annotations.jar</Link>
+    </None>
+    <None Include="..\..\External-Dependency-Info.txt" Pack="True" PackagePath="THIRD-PARTY-NOTICES.txt" />
   </ItemGroup>
+
   <ItemGroup>
-    <TransformFile Include="Transforms\EnumFields.xml" />
-    <TransformFile Include="Transforms\EnumMethods.xml" />
-    <TransformFile Include="Transforms\Metadata.xml" />
+    <EmbeddedReferenceJar Include="..\..\externals\guava.jar" />
+    <!-- <ReferenceJar Include="..\..\externals\animal-sniffer-annotations.jar" /> -->
```

Comment:
Should these comments be removed?

Reply:
Yeah they should be for good measure, though they aren't really needed since we're now just using EmbeddedReferenceJar and aren't producing any API bindings.  It's good to keep them uncommented for future releases if we do change this though, thanks!.